wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+NavigationBar/ConversationListViewController+NavigationBar.swift

```diff
@@ -143,6 +145,8 @@ extension ConversationListViewController: ConversationListContainerViewModelDele
             L10n.Localizable.ConversationList.Filter.OneOnOneConversations.title
         case (.collapsed, _):
             L10n.Localizable.List.title
+        case (.expanded, .folder):
+            L10n.Localizable.ConversationList.Filter.Folders.title
```

Comment:
optional: please move the added lines up so that all `.expanded` cases are grouped.

Reply:
Will fix.